lib/commons/utils/valid-input-type.js

```diff
@@ -0,0 +1,38 @@
+/* global axe */
+
+/**
+ * Returns array of valid input type values
+ * @method validInputTypes
+ * @memberof axe.commons.utils
+ * @instance
+ * @return {Array<Sting>}
+ */
+axe.utils.validInputTypes = function validInputTypes() {
```

Comment:
This should be assigned to axe.commons.utils, not to axe.utils. Also, I don't think this should be a function. It's inconsistent with how other thing in axe behave.